### DISCUSSION TITLE
How to contribute lyrics?

### DISCUSSION THREAD

**Discussion by NikunjK8** [tranxuanthang/lrcget#67] [General]
Is there a way i can add lyrics of a song in lrclib database if I'm not a developer? Or maybe suggest lyrics?

**Comment by tranxuanthang** [comment]
Yes, you definitely can! You also do not need an account or any kind of registration to contribute lyrics.

Look for Publish option in the LRCGET's lyrics editor:

![image](https://github.com/tranxuanthang/lrcget/assets/15942946/4b967725-cd55-4033-9fcd-b4b115315605)

**Comment by NikunjK8** [comment]
Got it. Thanks so much :)
